sdks/dart/client-dart-sdk/README.md

```diff
@@ -115,3 +115,58 @@ enum Readyness {
 }
 
 ```
+
+## Mobile API
+
+FeatureHub also has a mobile friendly API, that is intended to be used like a GET request and disconnect. The
+point here is that the Mobile devices radio does not stay on like it would with an EventSource. For Dart server 
+instances, client side apps (such as Flutter for Web, Linux, Windows or Mac OS X) - you should use the Event Source
+above. For Flutter for Android or iOS, please use this API.
+
+It is simple to use, you need to specify the host base url and the environment(s) that you wish to pull into your 
+application. Do not have features with the same names otherwise you will encounter issues with versioning.
```

Comment:
Why would someone have features with same names (or same keys?)? Is it something about mobile?

Reply:
Each SDK can be a completely different environment in a different application. So two applications could have the same keys.

Reply:
it does "names" maybe should be "keys"